utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -2,18 +2,25 @@ use log::{info, warn};
 use std::cell::RefCell;
 use std::cmp::max;
 use std::collections::{HashMap, HashSet};
-use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::atomic::AtomicUsize;
 use std::sync::{Arc, Mutex};
 use std::task::Poll;
 use std::time::Duration;
 use std::time::Instant;
 
 use futures;
 use futures::task::Context;
+use near_rust_allocator_proxy::allocator::{
+    current_thread_memory_usage, current_thread_peak_memory_usage, reset_memory_usage_max,
+    thread_memory_usage,
+};
 use once_cell::sync::Lazy;
 use std::pin::Pin;
 use strum::AsStaticRef;
 
+const MEBIBYTE: usize = 1024 * 1024;
```

Comment:
Do you mean "MEGABYTE"?

Reply:
No, the proper term is Mebibyte, also denoted as MiB. It's 1024*1024 bytes.
https://simple.wikipedia.org/wiki/Mebibyte#:~:text=A%20mebibyte%20is%20a%20unit,to%20the%20megabyte%20(MB).
I copied constant from https://docs.rs/size/0.1.2/size/constant.MEBIBYTE.html
Megabyte is 1000*1000 bytes.